pkg/cmd/root/help_topic.go

```diff
@@ -91,44 +91,96 @@ var HelpTopics = map[string]map[string]string{
 	"formatting": {
 		"short": "Formatting options for JSON data exported from gh",
 		"long": heredoc.Docf(`
-			Some gh commands support exporting the data as JSON as an alternative to their usual
-			line-based plain text output. This is suitable for passing structured data to scripts.
-			The JSON output is enabled with the %[1]s--json%[1]s option, followed by the list of fields
-			to fetch. Use the flag without a value to get the list of available fields.
-
-			The %[1]s--jq%[1]s option accepts a query in jq syntax and will print only the resulting
-			values that match the query. This is equivalent to piping the output to %[1]sjq -r%[1]s,
-			but does not require the jq utility to be installed on the system. To learn more
-			about the query syntax, see: <https://stedolan.github.io/jq/manual/v1.6/>
-
-			With %[1]s--template%[1]s, the provided Go template is rendered using the JSON data as input.
-			For the syntax of Go templates, see: <https://golang.org/pkg/text/template/>
-
-			The following functions are available in templates:
-			- %[1]sautocolor%[1]s: like %[1]scolor%[1]s, but only emits color to terminals
-			- %[1]scolor <style> <input>%[1]s: colorize input using <https://github.com/mgutz/ansi>
-			- %[1]sjoin <sep> <list>%[1]s: joins values in the list using a separator
-			- %[1]spluck <field> <list>%[1]s: collects values of a field from all items in the input
-			- %[1]stablerow <fields>...%[1]s: aligns fields in output vertically as a table
-			- %[1]stablerender%[1]s: renders fields added by tablerow in place
-			- %[1]stimeago <time>%[1]s: renders a timestamp as relative to now
-			- %[1]stimefmt <format> <time>%[1]s: formats a timestamp using Go's Time.Format function
-			- %[1]struncate <length> <input>%[1]s: ensures input fits within length
+		By default, the result of %[1]sgh%[1]s commands are output in line-based plain text format. Some commands support passing the --json flag, which converts the output to JSON format. Once in JSON, the output can be further formatted according to a required formatting string by adding either the --jq or --template flag. This is useful for selecting a subset of data, creating new data structures, displaying the data in a different format, or as input to another command line script.
+
+		The --json flag requires a comma separated list of fields which match the JSON field names from an API response. For a list of available fields, see: <https://docs.github.com/en/graphql/reference/>. Note that you must pass the --json flag and field names to use the --jq or --template flags.
```

Comment:
"For a list of available fields, see GraphQL reference" — this isn't correct, since our json fields do not necessarily have 1-1 mapping to our GraphQL schema (sometimes they are just made up). The previous documentation which you removed was correct:

> Use the flag without a value to get the list of available fields.

If that sentence wasn't clear, perhaps you can modify it to make it clearer?